src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea {
```

Comment:
The interface name `FigureArea` suggests a verb-like action which is not recommended for interfaces. Consider renaming the interface to something that represents a characteristic or ability, such as `AreaCalculatable` to follow the Java naming conventions.